sign up

AidaNajafi opened the pull request.

AidaNajafi committed:
sign up

alirezadoostimehr commented:
Please put the /Podcastino/venv folder in .gitignore.
Also add the requirements.txt file, you can make it with pip freeze (to see all the used libraries here.) data.

alirezadoostimehr committed:
Create .gitignore file.

alirezadoostimehr committed:
Untrack venv folder.

alirezadoostimehr committed:
Move files into main folder.

alirezadoostimehr committed:
Create requirements.txt file.

alirezadoostimehr committed:
Switch project to Postgresql.

Add docker compose file.

Update requirements.txt file.

Create .env file.

Update README.md file.

alirezadoostimehr committed:
Move Signup app tp auth to be used for login too.

alirezadoostimehr committed:
Minor fix.

alirezadoostimehr committed:
Add new migration file.

alirezadoostimehr committed:
Add login APIs.

alirezadoostimehr committed:
Add reset password api.

Add sending email settings.

alirezadoostimehr committed:
Add and run Ruff on project.

alirezadoostimehr committed:
Delete Podcastino/Podcastino/__pycache__ directory

alirezadoostimehr committed:
Delete Podcastino/venv/lib/python3.12/site-packages directory

alirezadoostimehr committed:
Delete Podcastino/SignUp directory
